blob/fileblob/fileblob.go

```diff
@@ -69,14 +54,28 @@ import (
 
 const defaultPageSize = 1000
 
-func init() {
-	blob.Register("file", func(_ context.Context, u *url.URL) (driver.Bucket, error) {
-		path := u.Path
-		if os.PathSeparator != '/' && strings.HasPrefix(path, "/") {
-			path = path[1:]
-		}
-		return openBucket(path, nil)
-	})
+// Scheme is the URL scheme conventionally used for the filesystem in a URLMux.
+const Scheme = "file"
```

Comment:
I'm not sure if this should be `file` or `fileblob`. If there is more structure for fileblobs, then I would tend to go with the second one, but I don't know this package well enough to say for sure.